templates/types/streaming/fastapi/pyproject.toml

```diff
@@ -36,4 +36,8 @@ ignore_missing_imports = true
 follow_imports = "silent"
 implicit_optional = true
 strict_optional = false
-disable_error_code = ["return-value", "import-untyped", "assignment"]
+disable_error_code = ["return-value", "assignment"]
+
+[[tool.mypy.overrides]]
```

Comment:
Cannot check for missing imports for all the packages; configure it to only check for missing imports in our code